### PR TITLE
Don't show archived projects as trending ever

### PR DESCRIPTION
**Pull request by tomquirk**
- archives project 209
- Updates trending projects query to _not_ include archived projects
- smol refactoring/renaming of that file.

### CODE REVIEW

File: src/hooks/Projects.ts [hunk 1]
```diff
@@ -152,6 +152,13 @@ export function useTrendingProjects(count: number) {
     first: count,
     orderBy: 'trendingScore',
     orderDirection: 'desc',
+    where: [
+      {
+        key: 'id',
+        value: ARCHIVED_SUBGRAPH_IDS,
+        operator: 'not_in',
+      },
+    ],
```
Comment (tomquirk) [inline src/hooks/Projects.ts:161]: This is the fix

File: src/locales/messages.pot [hunk 2]
```diff
@@ -71,7 +71,7 @@ msgstr ""
 msgid "90 days"
 msgstr ""
 
-msgid "<0/> Archived projects have not been modified or deleted on the blockchain, and can still be interacted with directly through the Juicebox contracts."
+msgid "<0/> Archived projects haven't been modified or deleted on-chain. They can still be interacted with directly through the Juicebox contracts."
```
Comment (tomquirk) [inline src/locales/messages.pot:74]: smol copy change

File: src/hooks/Projects.ts (hunk 1 above)
Comment (johnnyd-eth) [reply src/hooks/Projects.ts:161]: Clean.

**Review by johnnyd-eth — APPROVED** [review]
LGTM!

**Pull request merged**